src/data/packet-structure.json

```diff
@@ -5,6 +5,12 @@
     "Company logo on our website",
     "Distribute company swag at the event"
   ],
+  "base-mobile" : [
```

Comment:
i think it makes more sense for the text to not change between the two, so just stick w the og if it fits otherwise both can use this version